presenters/booking_presenter.rb

```diff
@@ -0,0 +1,52 @@
+class BookingPresenter
+  TICKETS_COUNT_LIMIT = 5
```

Comment:
Just an FYI - there's no need to freeze Floats, Ints & Symbols cause they're immutable by default